src/synthesize.jl

```diff
@@ -160,39 +159,74 @@ function synthesize(atm::ModelAtmosphere, linelist, λs::AbstractRange; metallic
 end
 
 """
-    get_absolute_abundances(metallicity, abundances, solar_abundances, solar_relative)
+    construct_abundances(metallicity, abundances)
 
-Calculate ``n_X/n_\\mathrm{total}`` for each element X given abundances in either [``X``/H] or 
-``A(X)`` form.  See [`synthesize`](@ref) for a detailed description of the arguments. Returns a 
-vector indexed by atomic number.
+Returns a 92 element vector containing abundances in ``A(X)`` (``\\log(X/H) + 12``) format for 
+elements from hydrogen to uranium.
+
+# Arguments
+You can provide either or both of:
+- `metallicity` (default: 0), i.e. [metals/H] is the ``\\log_{10}`` solar-relative abundance of elements heavier 
+   than He. It is overriden by `abundances`.  
+- `abundances` is a `Dict` mapping atomic numbers or symbols to [``X``/H] abundances.  (Set 
+  `solar_relative=false` to use ``A(X)`` abundances instead.) These override `metallicity`.
+
+# Keyword arguments
+- `solar_relative` (default: true): When true, interpret abundances as being in \\[``X``/H\\] 
+  (``\\log_{10}`` solar-relative) format.  When false, interpret them as ``A(X)`` abundances, i.e. 
+   ``A(x) = \\log_{10}(n_X/n_\\mathrm{H}) + 12``, where ``n_X`` is the number density of ``X``.
+   Note that abundances not specified default to the solar value, adjusted with `metallicity`, in
+   either case.
```

Comment:
On the remote chance you didn't think deeply about this (while you were reorganizing), this keyword argument seems like it could potentially be removed (since the user can directly modify the output). 

With that said, I expect that you already thought about this and decided that it was important to included this option to make things more convenient to users (which I think is a totally valid choice) - if that's the case just resolve this conversation.